include/znc/zncconfig.h.cmake.in

```diff
@@ -43,6 +43,9 @@
 #cmakedefine HAVE_TCSETATTR 1
 #cmakedefine HAVE_GETPASSPHRASE 1
 #cmakedefine HAVE_CLOCK_GETTIME 1
+#cmakedefine HAVE_GETHOSTNAME 1
```

Comment:
Please add ZNC_ prefix like I did with argon

The other lines should also gain that, but I don't really want to break compatibility

Reply:
No problem.

...I'm missing some context though, just for curiosity: what is argon, and what compatibility are you afraid to break?

Reply:
argon2 is password hashing library, see HAVE line above

I'd like to rename all these HAVE lines to start with ZNC, because when external modules are compiled, the headers are in /usr/include/znc, and these HAVE macros can conflict with whatever other libraries the module uses.

But the same modules also might depend on these spelling of these HAVE lines